api/trakt.go

```diff
@@ -1773,7 +1776,7 @@ func renderProgressShows(ctx *gin.Context, shows []*trakt.ProgressShow, total in
 			item.Info.Aired = airDate
 			item.Info.DateAdded = airDate
 			item.Info.Premiered = airDate
-			item.Info.LastPlayed = airDate
+			//item.Info.LastPlayed = airDate
```

Comment:
Leftovers? `LastPlayed` not needed?

Reply:
i did it for test since i read:
https://kodi.wiki/view/JSON-RPC_API/v13#Files.SetFileDetails

>[[Optional.String](https://kodi.wiki/view/JSON-RPC_API/v13#Optional.String) lastplayed = None] (Setting a valid lastplayed without a playcount will force playcount to 1.)

so i guess we should not set LastPlayed for all items.

although other docs
https://alwinesch.github.io/group__python__xbmcgui__listitem.html#ga0f1e91e1d5aa61d8dd0eac90e8edbf18
and
https://kodi.wiki/view/InfoLabels

does not say anything about LastPlayed.

anyway, i have not seen any difference when i removed set of LastPlayed.
maybe kodi was smart enough to ignore it.

but again - we should not set LastPlayed to airDate anyway, since it says
>Last play date of current playing Video, if it's in the database.

and we set to not yet played videos.

Reply:
@elgatito 

i did it for a test since i read:
https://kodi.wiki/view/JSON-RPC_API/v13#Files.SetFileDetails

>[[Optional.String](https://kodi.wiki/view/JSON-RPC_API/v13#Optional.String) lastplayed = None] (Setting a valid lastplayed without a playcount will force playcount to 1.)

so i guess we should not set LastPlayed for all items.

although other docs
https://alwinesch.github.io/group__python__xbmcgui__listitem.html#ga0f1e91e1d5aa61d8dd0eac90e8edbf18
and
https://kodi.wiki/view/InfoLabels

does not say anything about LastPlayed.

anyway, i have not seen any difference when i removed set of LastPlayed.
maybe kodi was smart enough to ignore it.

and again - we should not set LastPlayed to airDate anyway, since it says
>Last play date of current playing Video, if it's in the database.

and we set to not yet played videos. and anyway the date is fake (user have not played video at that date).

so i think we should delete this code to avoid confusion.

Reply:
In my understanding `lastPlayed` was about last air date, and not about when I was watching it last time.

But all the docs are saying that `lastPlayed` is about playback. Don't know how it worked before if that field was always set.